conf.go

```diff
@@ -51,6 +52,7 @@ type Result struct {
 	AlexaRank   int         `json:"alexa_rank,omitempty"`
 	Status      string      `json:"status,omitempty"`
 	Error       string      `json:"error,omitempty"`
+	Timestamp   *time.Time  `json:"timestamp,omitempty"`
 	Data        interface{} `json:"data,omitempty"`
 }
```

Comment:
Let's change the output format to match ZGrab.

If you're not familiar with how to do this "cleanish" in Golang, you're going to have to write a custom `func (r *Result) MarshalJSON() ([]byte, error)` function to handle the formatting. I'd probably put this in a new file called `json.go`. You'll need to create a private auxillary type that has a `Timestamp string`, and manually convert using `time.Format`. Then just `return json.Marshal(&auxStruct)`.